objects/src/builder/mod.rs

```diff
@@ -230,11 +230,25 @@ impl<T: Rng> AccountBuilder<T> {
         self
     }
 
+    pub fn add_assets<I: IntoIterator<Item = Asset>>(mut self, assets: I) -> Self {
+        for asset in assets.into_iter() {
+            self.assets.push(asset);
+        }
+        self
+    }
+
     pub fn add_storage_item(mut self, item: StorageItem) -> Self {
         self.storage_builder.add_item(item);
         self
     }
 
+    pub fn add_storage_items<I: IntoIterator<Item = StorageItem>>(mut self, items: I) -> Self {
+        for item in items.into_iter() {
+            self.storage_builder.add_item(item);
+        }
+        self
+    }
```

Comment:
Added these versions because this is a owning builder, doing this in the caller makes for ugly code.